helical/models/geneformer/geneformer_utils.py

```diff
@@ -4,13 +4,14 @@
 import pickle as pkl
 # imports
 import logging
-
+import numpy as np
+from sklearn.metrics import accuracy_score
 import torch
 from tqdm.auto import trange
 import re
 import torch
 from transformers import (
-    BertForMaskedLM,
+    BertForMaskedLM,get_scheduler
```

Comment:
Do we still use this function?

Reply:
get_scheduler has been removed

Reply:
But we do still use it as a scheduler loader